cli/types/cypress.d.ts

```diff
@@ -792,7 +819,35 @@ declare namespace Cypress {
     clearCookies(options?: CookieOptions): Chainable<null>
 
     /**
-     * Clear data in local storage.
+     * Get local storage for all origins.
+     *
+     * @see https://on.cypress.io/getalllocalstorage
+     */
+    getAllLocalStorage(options?: Partial<Loggable>): Chainable<StorageByOrigin>
```

Comment:
Shouldn't this return an array?
```suggestion
    getAllLocalStorage(options?: Partial<Loggable>): Chainable<Array<StorageByOrigin>>
```